### ISSUE TITLE
Gender Selection

### ISSUE THREAD

**Issue by Xarcell** [elkarte/Elkarte#1322]
Allow the choice between 3 genders. Male, Female, and Intersex. several countries recognize a third gender like Intersex, such as: Australia, Germany, India, Nepal, and Bangladesh.

We are talking about those that are born with both characteristic's or genitally of male/female. Not those that are transgender, who switch genders or cross-gender.

I would also like to see gender be directly tied to membergroups, so that permissions can be placed on user's based on their gender.

Of course, then you would have make it so that a user's gender can only be "changed" by the admin or global moderator.

**Comment by Vekseid** [comment]
Intersex isn't a gender, it's a physiological condition.

Genders could be male/female/both/none/other/undisclosed

Since even Facebook supports this these days, I think this can be supported more directly (if only by permitting admins to define more fields).

**Comment by Spuds** [comment]
I wonder if that standard field should be removed an just left for the admin to use a custom profile field instead then.  Could seed it with your suggestion and and admin can add / remove as desired.  Not sure what to do in the gender icons though.

**Comment by Xarcell** [comment]
@Spuds if you make gender a custom profile field, then so should all other fields. Which might not be a bad idea when you think about it. What would be the pros and cons of it? Pros: Leaner code, less maintenance, smaller package, ... Cons: people bitching it's not there by default, people bitching that they don't understand how to create profile fields, people bitching that it was changed...

If it were me, all profile fields would have to be created via admin. However, how would this effect mods that check for gender?

**Comment by Spuds** [comment]
@Xarcell good to see you around !

Honestly I'm not a big fan of the default profile fields and have long thought about removing them all as they are just kind of messy in the code.  They are nice for noobs who can find the custom fields intimidating on first use. 

I'd think we could just preload additional "default" custom fields at the start and cover whats there, or even have a small library of templates that gets json fetched that you could select from.  

Probably right about addons that use gender or any of the standard fields, the ones that remain are kind of dug in.  Perhaps the use of reserved names (ugh) for them so at least addons could find the ones that were typically around,  have not looked at that code area in a bit so that may not be necessary.

**Comment by Vekseid** [comment]
@Xarcell why would there be cons? You could just have a default set of custom fields.

@Spuds custom fields should already be able to reference images, shouldn't they?

**Comment by Vekseid** [comment]
@Spuds @emanuele45 I noticed you moved this to 2.0 but gender is something I kindof want to take care of. Like, this weekend. >_>

Would you object to me removing the current hardcoded set of profile fields and making custom defaults? Including a properly complete gender selection. Font Awesome has a complete icon set for genders after all.

**Comment by Spuds** [comment]
I don't have any objections.   I *think* why it may have been moved was the thought that all of the "built in" fields should be moved to custom profile fields where we simply install the proper custom template for use.  This is what pretty much happened with all of the IM fields in 1.0

**Comment by emanuele45** [comment]
I moved it to beta because 6 months ago the release date seemed close, then the mistake happened and I still have to fix what I broke... :blush: 
The idea is for sure to convert everything (possible) to profile fields. The gender is a bit of a special case because it is associated to the statistics page and **may** have some more implications (I never check too deeply so I may be completely wrong) in the conversion.

If you want to work on if, feel free to! :D
